### PR TITLE
Update dockerfile to copy the Config file into the docker image

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by geoffjentry** [comment]
@scottfrazer This looks right to us, but can you take a look and double check our docker-fu?

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ RUN ["/bin/bash", "-c", "/cromwell/docker/install.sh /cromwell"]
 RUN mkdir /etc/service/cromwell
 ADD docker/run.sh /etc/service/cromwell/run
 
+COPY src/main/resources/reference.conf /etc/cromwell.conf
+
```
Comment (scottfrazer) [inline Dockerfile]: Each Docker command you have adds a new layer to the image, so it's generally a good idea to limit the number of layers.  Try adding the `cp` to the RUN command above:

```docker
RUN mkdir /etc/service/cromwell && cp /cromwell/src/main/resources/reference.conf /etc/cromwell.conf
```

Comment (scottfrazer) [inline Dockerfile]: Actually the ADD command above could also be folded into the RUN line too:

```
RUN mkdir /etc/service/cromwell && 
         cp /cromwell/src/main/resources/reference.conf /etc/cromwell.conf &&
         cp /cromwell/docker/run.sh /etc/service/cromwell/run
```

Comment (geoffjentry) [inline Dockerfile]: Cool thanks, let me give this a whirl

**Comment by geoffjentry** [comment]
@scottfrazer Updated w/ your suggestions, appears to work fine in docker

**Pull request merged**